api_app/script_analyzers/yara_repo_downloader.sh

```diff
@@ -60,7 +60,10 @@ rm generic_anomalies.yar general_cloaking.yar thor_inverse_matches.yar yara_mixe
 
 # Download rules for quark-engine analyzer
 cd /opt/deploy
-svn export https://github.com/quark-engine/quark-engine/tags/v20.08/quark/rules quark-rules
+git clone https://github.com/quark-engine/quark-rules quark-rules
+# this is done to lock the version since the repo does not have tags.
+cd quark-rules
+git reset --hard b2cca423ed3c0786e6e41b2cb277676ed66d7854
 
```

Comment:
You went inside the directory and didn't come out and that's the reason why dnstwist can't find it's dictionary directory.

**Fix**: Just Insert `cd ..` here

Reply:
You are absolutely right, will fix on monday!